clipper_admin_v2/clipper_admin/clipper_admin.py

```diff
@@ -0,0 +1,488 @@
+from __future__ import absolute_import, division, print_function
+import logging
+import docker
+import requests
+import json
+import os
+from .container_manager import CONTAINERLESS_MODEL_IMAGE
+
+DEFAULT_LABEL = ["DEFAULT"]
+
+logger = logging.getLogger(__name__)
+
+
+class ClipperException(Exception):
+    pass
+
+
+def start_clipper(cm):
```

Comment:
Any reason why this isn't encapsulated in a class? Seems weird to have the end user maintain the reference to the `ContainerManager` rather than have that resource managed by a class which exposes this public API.